css/global.css

```diff
@@ -0,0 +1,10 @@
+.org-organization-admin-pages-entrypoint-card__card,
+.feed-identity-module__widgets,
+.artdeco-toast-item--visible,
+.feed-new-update-pill__new-update-button,
+aside > section.artdeco-card.ember-view.relative.break-words.pb3.mt2,
```

Comment:
This hides any sections with the listed class names within an aside - which is both of the sidebar sections related to people suggestions.